website/src/css/custom.css

```diff
@@ -0,0 +1,161 @@
+:root {
+  --ifm-color-primary: #9e8400;
```

Comment:
Compared to our previous primary color `#f5da55: hsv(50, 65.3%, 96.1%)`, the new color boosted the saturation and lowered the value `hsv(50, 100%, 62%)` to increase the contrast to the white background.